src/sky.cpp

```diff
@@ -519,13 +560,16 @@ void Sky::render()
 			}
 		}
 
-		// Draw far cloudy fog thing below east and west horizons
+		driver->setMaterial(m_materials[1]);
+
+		// Draw lower blend fog band
 		for (u32 j = 0; j < 2; j++) {
-			video::SColor c = cloudyfogcolor;
-			vertices[0] = video::S3DVertex(-1, -1.0,  -1, 0, 0, 1, c, t, t);
-			vertices[1] = video::S3DVertex( 1, -1.0,  -1, 0, 0, 1, c, o, t);
-			vertices[2] = video::S3DVertex( 1, -0.02, -1, 0, 0, 1, c, o, o);
-			vertices[3] = video::S3DVertex(-1, -0.02, -1, 0, 0, 1, c, t, o);
+			video::SColor c = m_bgcolor;
+			video::SColor c_blend = m_bgcolor_alpha;
+			vertices[0] = video::S3DVertex(-1, -0.2,  -1, 0, 0, 1, c, t, t);
+			vertices[1] = video::S3DVertex( 1, -0.2,  -1, 0, 0, 1, c, o, t);
+			vertices[2] = video::S3DVertex( 1, -0.1, -1, 0, 0, 1, c_blend, o, o);
+			vertices[3] = video::S3DVertex(-1, -0.1, -1, 0, 0, 1, c_blend, t, o);
```

Comment:
-0.1 instead of -0.02 is wrong because this quad is what cuts off the sun and moon as they disappear over the horizon, so you've moved the horizon to -0.1 which is significantly below the true horizon at 0.0. This will cause more visual disorientation.
Needs to be somewhere between -0.02 and 0.0, preferably -0.02.

Reply:
-0.02 is the same place as the original horizon you mentioned starts.

Reply:
Yes, but i can't see the relevance of that because you've moved the effective horizon to around -0.15.

Reply:
We have the false horizontal horizon lowered, so that if you were on top of a mountain or hill, the horizon still exists, but not looking flat at the camera. But standing at sea level, it's clear that the sun or moon sets as normal. It just masks the lack of an actual curvature or fixed Y level horizon.

Reply:
Eh, it's fine. Who cares other than the people demanding an *ultra realistic game with brown tinted shaders*. I mean, it looks fancy, and that's what I aimed for.

Reply:
It has nothing to do with ultra realism, as you know i would be the last to insist on ultra realism :)
In an abstract MT world the horizon is still unarguably at a particular elevation, this is indpendent of realism.
You seem to now be arguing for sloppiness, we care because we care about MT and it's quality.
Fanciness alone is not enough, there also has to be quality and precision :)

Reply:
See comments in the thread, the -0.02 used in master is well chosen to account for an average player altitude of y = 500. Please move the horizon back to -0.02, as in, the middle point of the gradient that hides the sun and moon should be at -0.02.